packages/dev-console/src/ExtensionServerClient/APIClient.ts

```diff
@@ -0,0 +1,15 @@
+export class APIClient implements ExtensionServer.API.Client {
+  constructor(public url: string) {}
+
+  async extensions(): Promise<ExtensionServer.API.ExtensionsResponse> {
+    const response = await fetch(`${this.url}/extensions`);
+    const dto: ExtensionServer.API.ExtensionsResponse = await response.json();
```

Comment:
what does dto stand for?

Reply:
DTO stands for `data transfer object`. It's a common term for an object that carries data but has no logic.